src/test/java/io/vertx/core/parsetools/FakeStream.java

```diff
@@ -89,4 +93,12 @@ void fail(Throwable err) {
   void end() {
     endHandler.handle(null);
   }
+
+  public int getPauseCount() {
```

Comment:
rather `pauseCount()` and `resumeCount()`

Reply:
done